frontend/src/old-pages/Configure/Configure.tsx

```diff
@@ -301,22 +308,26 @@ function Configure() {
   useWizardSectionChangeLog()
 
   return (
-    <Layout contentType="form">
-      <StopDialog clusterName={clusterName} />
-      <SpaceBetween direction="vertical" size="l">
-        <BreadcrumbGroup
-          items={[
-            {text: 'Clusters', href: '#clusters'},
-            // @ts-expect-error TS(2741) FIXME: Property 'href' is missing in type '{ text: string... Remove this comment to see the full error message
-            {text: editing ? 'Update' : 'Create'},
-          ]}
+    <Layout
+      contentType="form"
+      breadcrumbs={
+        <Breadcrumbs
           onClick={clearStateAndCloseWizard}
+          pages={[
+            breadcrumbItemFromSlug('clusters'),
+            breadcrumbItem(editing ? 'Update' : 'Create'),
+          ]}
         />
+      }
+    >
+      {/*<Container >*/}
+      <StopDialog clusterName={clusterName} />
+      <SpaceBetween direction="vertical" size="l">
         <SpaceBetween direction="horizontal" size="s">
           <SideNav />
           <div style={{minWidth: '800px', maxWidth: '1000px'}}>
             <SpaceBetween direction="vertical" size="s">
-              <Box>
+              <Box margin={{top: 'l'}}>
```

Comment:
todo: avoid using inline style (for the same reason as `useMemo`): the styles can be extracted outside the component with a simple `const`